DQM/L1TMonitor/src/L1TRPCTPG.cc

```diff
@@ -123,7 +123,7 @@ void L1TRPCTPG::analyze(const Event& e, const EventSetup& c)
   vector<L1MuGMTReadoutRecord> gmt_records = gmtrc->getRecords();
   vector<L1MuGMTReadoutRecord>::const_iterator RRItr;
   
-  static int nRPCTrackBarrel, nRPCTrackEndcap , nDTTrack, nCSCTrack;
+  int nRPCTrackBarrel, nRPCTrackEndcap , nDTTrack, nCSCTrack;
```

Comment:
With the values being reset each call, it definitely looks like there was no need for these to be static.